Formula/elasticsearch.rb

```diff
@@ -115,13 +120,41 @@ def plist
     server.close
 
     system "#{bin}/elasticsearch-plugin", "list"
-    pid = "#{testpath}/pid"
+    pid = testpath/"pid"
     begin
       system "#{bin}/elasticsearch", "-d", "-p", pid, "-Epath.data=#{testpath}/data", "-Ehttp.port=#{port}"
       sleep 10
       system "curl", "-XGET", "localhost:#{port}/"
     ensure
-      Process.kill(9, File.read(pid).to_i)
+      Process.kill(9, pid.read.to_i)
+    end
+
+    require "socket"
```

Comment:
You seem to have required `socket` twice?

Reply:
I pushed.